From c599480bdec0ed57922d100dceb5c94d04755694 Mon Sep 17 00:00:00 2001
From: Quirin Gylstorff <quirin.gylstorff@siemens.com>
Date: Wed, 15 Feb 2023 10:47:38 +0100
Subject: [PATCH 2/2] d/control: remove :native from python3-shtab

This fixes the cross build for qemu-arm and qemu-arm64.

Signed-off-by: Quirin Gylstorff <quirin.gylstorff@siemens.com>
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 8b92019..53e2b66 100644
--- a/debian/control
+++ b/debian/control
@@ -9,7 +9,7 @@ Build-Depends: debhelper,
  help2man,
  libpci-dev,
  pkg-config,
- python3-shtab:native,
+ python3-shtab,
  zlib1g-dev
 Standards-Version: 4.6.1
 Homepage: https://github.com/siemens/efibootguard
-- 
2.39.2

